static/src/js/components/EDSCImage/__tests__/EDSCImage.test.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable testing-library/no-node-access */
```

Comment:
Why are these rules ignored? Does the eslintrc rules cover them, or do they need to be addressed?

If they do have to be ignored, we should use the `// eslint-disable-next-line` syntax only on the lines that need it. In general this means we don't accidentally write code in a file that gets ignored but could have been fixable